public/lessons/react/conditionally_rendering_components.md

```diff
@@ -205,3 +205,14 @@ class Page extends React.Component {
 ```
 
 [Example CodePen](https://codepen.io/Dangeranger/pen/BPvEqv?editors=0010)
+
+# Lab: Modal Display
```

Comment:
This is a good lab, students seem to be a bit confused at first when modals are brought up, so this should help clear up some of that confusion